test/images/image-util.sh

```diff
@@ -141,10 +141,10 @@ push() {
 # This function is for building the go code
 bin() {
   local arch_prefix=""
-  if [[ "${ARCH}" == "arm" ]]; then
+  if [[ "${ARCH:?}" == "arm" ]]; then
     arch_prefix="GOARM=${GOARM:-7}"
   fi
-  for SRC in $@;
+  for SRC in "$@";
```

Comment:
```
In ./test/images/image-util.sh line 147:
  for SRC in $@;
             ^-- SC2068: Double quote array expansions to avoid re-splitting elements.
```